reverse-strings/tools/reverse.go

```diff
@@ -0,0 +1,16 @@
+package tools
+
+func Reverse(in string) string {
+	tmp := make([]rune, len(in))
+	amt := 0
+	for _, v := range in {
+		tmp[amt] = v
+		amt++
+	}
+	tmp = tmp[:amt]
+	reversed := make([]rune, amt)
```

Comment:
Nice work! But there is a better way to reverse a string without allocation of 2 slices
Try to find out what it is